21_pattern_examples.md

```
@@ -256,6 +256,152 @@ int main() {
+    for (int i = n - 1; i >= 1; i--) {
+        for (int s = 1; s <= n - i; s++) printf(" ");
+        for (int j = 1; j <= 2*i - 1; j++) printf("*");
+        printf("\n");
+    }
+
+    return 0;
+}
+```
+
+**Expected output:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)